proto/ardaos/spv/module/module.proto

```
@@ -1,14 +1,14 @@
 syntax = "proto3";
-package ardaos.ardaos.module;
+package ardaos.spv.module;
 
 import "cosmos/app/v1alpha1/module.proto";
```

Buf (1.55.1)

import "cosmos/app/v1alpha1/module.proto": file does not exist

(COMPILE)